stable/admin/templates/statefulset.yaml

```diff
@@ -112,6 +112,12 @@ spec:
       {{- end }}
         args:
           - "nuoadmin"
+          {{- if .Values.admin.evicted}}
+          {{- if .Values.admin.evicted.servers}}
```

Comment:
As a stylistic choice, I prefer using {{- with ... }} in place of {{- if ... }}  - and particularly a nested {{- if }} as the {{with}} avoids the need to dereference the desired value twice.

So something like:
{{- if Values.admin.evicted }}
{{- with Values.admin.evicted.servers }}
--evicted-servers
{{ join . "," }}
{{- end}}
{{- end}}

I think it's more clear that the {{- if}} is a guard against an empty structure, and the {{- with}} encloses some conditional rendering.

But a minor point...